app/src/main/java/com/infomaniak/mail/MatomoMail.kt

```diff
@@ -183,7 +185,7 @@ object MatomoMail : MatomoCore {
     }
 
     fun Fragment.trackOnBoardingEvent(name: String) {
-        trackEvent("onBoarding", name)
+        trackEvent("onboarding", name)
```

Comment:
Do we want to change the category name after using it for so long?

Reply:
yes

Reply:
why here, compared to other times where we don't?

Reply:
Because we are dissociated with iOS here, which isn't good 
And changing category has less impact than changing event names because we make our queries on event names
Also (and the fact that it is needed to do it fast now), we are entering a new year, and as we query on a full year time, it isn't a problem if it's different from the past year